packages/components/src/radio-group/types.ts

```diff
@@ -0,0 +1,39 @@
+export type RadioGroupProps = {
+	/**
+	 * Labels the value of the selected `Radio` element.
+	 */
+	label: string;
+	/**
+	 * The `value` of the `Radio` element which should be selected.
+	 * Indicates controlled usage of the component.
+	 */
+	checked?: string | number;
+	/**
+	 * The value of the radio element which is initally selected.
```

Comment:
Just spotted a minor typo as I was scanning through the types:

```suggestion
	 * The value of the radio element which is initially selected.
```